src/libraries/Native/Unix/Common/pal_utilities.h

```diff
@@ -132,3 +132,5 @@ inline static int32_t SizeTToInt32(size_t value)
     assert(value <= INT_MAX);
     return (int32_t)value;
 }
+
+#define min(x, y) (x) < (y) ? (x) : (y)
```

Comment:
This is a double-evaluating version of this macro.  It's probably best to not define it at all.